app/controllers/sessions_controller.rb

```diff
@@ -41,7 +43,7 @@ def lookup_login
                   set_login_info(username_or_email: @handler_result.outputs.username_or_email,
                                  names: @handler_result.outputs.names,
                                  providers: @handler_result.outputs.providers)
-                  redirect_to :authenticate
+                  render :authenticate
```

Comment:
I was doing `redirect_to` b/c I wanted the URL to change.  Not the best use, I know... but having `lookup_login` in the URL seems odd?